src/sql/workbench/contrib/notebook/browser/cellViews/textCell.component.html

```diff
@@ -4,7 +4,7 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 -->
-<div style="width: 100%; height: 100%; display: flex; flex-flow: column" (mouseover)="hover=true" (mouseleave)="hover=false">
+<div tabindex="0" style="width: 100%; height: 100%; display: flex; flex-flow: column" (mouseover)="hover=true" (mouseleave)="hover=false" (focus)="updateActiveCell()">
```

Comment:
This should likely have an aria-label if it's going to be tab-focusable now. Make sure you're checking with a screen reader that it's announcing something useful (like "text cell #" where number is the index in the Notebook)